src/layouts/sidebar-sidecar/components/table-of-contents/use-active-section.ts

```diff
@@ -123,7 +123,7 @@ export function useActiveSection(
         }
       },
       {
-        rootMargin: `-${getFullNavHeaderHeight()}px 0% -60% 0%`,
+        rootMargin: `-${getFullNavHeaderHeight()}px 0% -40% 0%`,
```

Comment:
This tweaks the intersection area to try to catch headings even if they're lower on the page. It makes a noticeable difference on some key pages, such as [/vault](https://dev-portal-git-zsrefine-footer-hashicorp.vercel.app/vault).

Reply:
This is a great tweak.